src/script/auth/page/ConversationJoin.tsx

```diff
@@ -21,64 +21,92 @@ import React, {useEffect, useState} from 'react';
 
 import type {RegisterData} from '@wireapp/api-client/lib/auth';
 import {BackendErrorLabel} from '@wireapp/api-client/lib/http';
-import {FormattedMessage, useIntl} from 'react-intl';
+import {useIntl} from 'react-intl';
 import {connect} from 'react-redux';
-import {Navigate} from 'react-router-dom';
 import {AnyAction, Dispatch} from 'redux';
 
 import {UrlUtil} from '@wireapp/commons';
 import {
-  ArrowIcon,
   Button,
+  Column,
+  Columns,
+  Container,
   ContainerXS,
   Form,
+  H1,
   H2,
   Input,
   InputBlock,
-  InputSubmitCombo,
-  Link,
-  RoundIconButton,
-  Small,
-  Text,
+  Loading,
+  Muted,
+  QUERY,
+  useMatchMedia,
 } from '@wireapp/react-ui-kit';
 
 import {noop} from 'Util/util';
 
+import {ConversationJoinFull, ConversationJoinInvalid} from './ConversationJoinInvalid';
 import {EntropyContainer} from './EntropyContainer';
+import {Login} from './Login';
+import {Page} from './Page';
 
 import {Config} from '../../Config';
 import {conversationJoinStrings} from '../../strings';
 import {AppAlreadyOpen} from '../component/AppAlreadyOpen';
-import {RouterLink} from '../component/RouterLink';
 import {UnsupportedBrowser} from '../component/UnsupportedBrowser';
 import {WirelessContainer} from '../component/WirelessContainer';
 import {EXTERNAL_ROUTE} from '../externalRoute';
-import {actionRoot as ROOT_ACTIONS} from '../module/action/';
+import {actionRoot as ROOT_ACTIONS} from '../module/action';
 import {ValidationError} from '../module/action/ValidationError';
 import {bindActionCreators, RootState} from '../module/reducer';
 import * as AuthSelector from '../module/selector/AuthSelector';
 import * as ConversationSelector from '../module/selector/ConversationSelector';
 import * as SelfSelector from '../module/selector/SelfSelector';
-import {QUERY_KEY, ROUTE} from '../route';
+import {QUERY_KEY} from '../route';
 import * as AccentColor from '../util/AccentColor';
 import {parseError, parseValidationErrors} from '../util/errorUtil';
-import * as StringUtil from '../util/stringUtil';
 
 type Props = React.HTMLProps<HTMLDivElement>;
 
+const Separator = () => {
+  const isMobile = useMatchMedia(QUERY.mobile);
+  const Line = () => (
+    <div
+      style={{
+        flex: 1,
+        height: '1px',
+        backgroundColor: '#696c6e',
+        minWidth: '24rem',
+      }}
+    ></div>
+  );
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        maxWidth: `${!isMobile ? '4rem' : '100%'}`,
+        justifyContent: 'center',
+        transform: `rotate(${!isMobile ? '90' : '0'}deg)`,
+        marginLeft: `${!isMobile ? '' : '16px'}`,
+      }}
```

Comment:
let's extract that to a `style` file and put it in the `css` property (not `style`)